posthog/models/team/team.py

```diff
@@ -184,6 +184,7 @@ class Team(UUIDClassicModel):
     )
     session_recording_linked_flag: models.JSONField = models.JSONField(null=True, blank=True)
     session_recording_network_payload_capture_config: models.JSONField = models.JSONField(null=True, blank=True)
+    session_recording_config: models.JSONField = models.JSONField(null=True, blank=True)
```

Comment:
nitpicking... should it be `session_replay_config`? (am open to be convinced not :))

Reply:
Nope, think you're right